tests/test_functools_utils.py

```diff
@@ -30,3 +32,6 @@ def method_c(self, value):
     assert test_obj.method(A()) == "a_or_b"
     assert test_obj.method(B()) == "a_or_b"
     assert test_obj.method(C()) == "c"
+    # can't
```

Comment:
can't?

Reply:
no idea :rofl: 
I imagine I had a thought.

Reply:
Half a thought.  I've completed it now :smile: